source/Firmata/UwpFirmata.h

```diff
@@ -321,13 +355,18 @@ public ref class UwpFirmata sealed
 	//when used with std::bind, this allows the Firmata library to invoke the function in the standard way (non-member type) while we redirect it to an object reference
 	static inline
 	void
-	analogInvoke(
+	stringInvoke(
 		UwpFirmata ^caller,
-		uint8_t pin_,
-		int value_
+		uint8_t *string_data
 	)
 	{
-		caller->AnalogValueEvent( caller, ref new CallbackEventArgs( pin_, value_ ) );
+		size_t len = strlen( reinterpret_cast<char *>( string_data ) ) + 1;
+		size_t wlen = len * sizeof( wchar_t );
+		wchar_t *wstr_data = (wchar_t *) malloc( wlen );
```

Comment:
Good catch, I was just moving this around and didn't pay any attention to it. I'll look back for more of these.